articles/resources/relnotes.md

```diff
@@ -25,6 +25,7 @@ For update instructions, please refer to the [update guide](xref:microsoft.quant
 - Added an [ANTLR grammar for Q#](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/5_Grammar) to the Q# language specification.
 - Updated the [`Microsoft.Quantum.Preparation` namespace](xref:Microsoft.Quantum.Preparation) to be more consistent with style guide and API design principles, and to support purified mixed states with additional data (see [proposal](https://github.com/microsoft/QuantumLibraries/issues/344) and [review notes](https://github.com/microsoft/QuantumLibraries/blob/main/Design/meetings/2020/api-design-2020-11-05.md)). https://github.com/microsoft/QuantumLibraries/pull/212, https://github.com/microsoft/QuantumLibraries/pull/322, https://github.com/microsoft/QuantumLibraries/pull/375, https://github.com/microsoft/QuantumLibraries/pull/376
 - Parentheses around repeated call expressions are now optional: `(Foo(x))(y)` may be written as `Foo(x)(y)`.
+- If users have .Net5 or upgraded to VS16.8, they will be prompt to install .Net3.1 to continue to work with Q#.
```

Comment:
I would suggest we talk specifically about the extensions here, rather than Q# as a whole.
```suggestion
- Users of the Visual Studio or Visual Studio Code extensions who have installed .NET 5 or Visual Studio 16.8 may be prompted to install .NET Core 3.1 to continue to work with the extensions.
```